cpp/src/io/parquet/experimental/page_index_filter.cu

```diff
@@ -693,18 +693,17 @@ std::vector<std::vector<bool>> aggregate_reader_metadata::compute_data_page_mask
   cudf::column_view row_mask,
   cudf::host_span<std::vector<size_type> const> row_group_indices,
```

Comment:
for some reason I can't comment on earlier changes in the file, so leaving a (very important) note here - line 226 should be restored.

Reply:
looks like I was reviewing an old version, don't see this now